src/styles/blocks/header.scss

```diff
@@ -0,0 +1,59 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.logo-link {
+  display: flex;
+  cursor: pointer;
+
+  @include transition;
+}
+
+.nav__links {
+  list-style: none;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin: 0;
+  padding: 0;
+
+  .nav__link-item {
+    &:not(:last-child) {
+      margin-right: 20px;
```

Comment:
The margin-right value should also follow the multiple of 4 rule. Use `margin-right: 16px;` or `margin-right: 24px;` instead of `margin-right: 20px;`.